src/components/ProgressionChecker/RadialBarChart.js

```diff
@@ -0,0 +1,48 @@
+import React from "react";
+import { RadialBar } from "@ant-design/charts";
+
+function RadialBarChart() {
+  var data = [
```

Comment:
Data is good. Keep it in a separate file or make an async call which simulates a real API call and return this data there.